src/helpers/GeographyHelpers.js

```diff
@@ -1,23 +1,20 @@
-import _ from 'lodash'
+import _ from 'lodash';
 // helper functions for parsing or formatting geographic data
 
 //attribute names to show on screen
 export const displayRegionAttributeNames = {};
 
-// for temporary use while only some data is available.
-const upper_fraser_watersheds = [
-    'Lower Chilako River', 'Lower Salmon River', 'Quesnel River',
-    'Salmon River', 'Cottonwood River', 'Euchiniko River', 
-    'Herrick Creek', 'Muskeg River', 'Nazko River', 'Morkill River', 
-    'Cariboo River', 'Bowron', 'Blackwater River', 'Willow River', 
-    'Tabor River', 'Narcosli Creek', 'Upper Fraser River', 
-    'Chilako River', 'McGregor River', 'Stuart River'
-    ];
 
-export function filterRegions(regions) {
-    return _.filter(regions, region => {
-    return(_.includes(upper_fraser_watersheds, region.name)); 
-    })
+// Is point a valid WKT point?
+export function validPoint(point) {
+    const p = JSON.parse(point)
+    return p 
+        && _.has(p, 'type')
+        && p.type === "Point"
+        && _.has(p, 'coordinates') 
+        && p.coordinates.length === 2 
+        && _.isNumber(p.coordinates[0])
+        && _.isNumber(p.coordinates[1]);
```

Comment:
And the series of `&&`s here can become `_.isNumber(p?.coordinates?.[0]) && _.isNumber(p?.coordinates?.[1])`